vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridViewSortingIT.java

```diff
@@ -184,6 +189,26 @@ public void gridWithSorting_multiSortPriorityAppend() {
                 QuerySortOrder.asc("firstName").thenDesc("age").build(), true);
     }
 
+    @Test
```

Comment:
nit: since the change just adds a bit of server-side logic, I assume it could be tested with a couple of unit tests. IT feels like overkill for the purpose if there's no flow-components client-side logic involved.

Reply:
Ah ok got it. I've removed the ITs and added unit tests now